spec/italian_food_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'italian_menu'
+
+describe ItalianMenu do
+  subject(:italian_menu) {described_class.new}
+  let(:lunch_desserts) {{ Desserts: 'Price', Pannacotta: 7, Semi_fresso: 7, Tiramisu: 10}}
+  let(:dinner_starters) {{ Starters: 'Price', Calamari: 8, Olives: 4, Panzerottini: 7, Bruschetta: 6, Gamberi: 9, Insalata: 7 }}
```

Comment:
Metrics/LineLength: Line is too long. [128/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)